app/pathogen/arbovirus/visualizations/page.tsx

```diff
@@ -26,28 +27,22 @@ export default function VisualizationsPage() {
   return (
     <div className="w-screen overflow-y-hidden grid grid-cols-12 grid-rows-2 -my-4 -ml-4 h-full-screen">
       <Filters className="col-span-2 row-span-2 overflow-y-scroll p-4 border-black border-r-2 h-full" />
-      <div className="flex-col flex h-full overflow-y-scroll col-span-10 row-span-2">
-        <div className="flex pt-4 mb-4">
-          <h3 className="w-full text-center text-lg">
-            {visualizationInformation.displayName}
-          </h3>
-          <button 
-            className="pr-8"
-            hidden={!referrerRoute || !isSafeReferrerLink(referrerRoute)}
-            onClick={() => {
-              if(referrerRoute && isSafeReferrerLink(referrerRoute)) {
-                router.push(referrerRoute)
-              }
-            }}
-            aria-label="Close Visualization"
-          >
-            <X />
-          </button>
-        </div>
-        <div className="flex-1">
-          {visualizationInformation.renderVisualization()}
-        </div>
-      </div>
+      <RechartsVisualization
```

Comment:
On fullscreen Chrome appears to occasionally download it with the scroll bars showing. I think this is because the overflow is set on the graph and not some kind of parent component. Anyways, I'm going to make myself a ticket and come back to this. There's some other visualization stuff I want to hit then as well and this PR is big enough lol